hack/ci-deploy.sh

```diff
@@ -84,9 +89,13 @@ kubectl -n=scylla-manager wait --timeout=5m --for='condition=Available=True' scy
 kubectl -n scylla-manager rollout status --timeout=5m deployment.apps/scylla-manager
 kubectl -n scylla-manager rollout status --timeout=5m deployment.apps/scylla-manager-controller
 
-kubectl -n haproxy-ingress rollout status --timeout=5m deployment.apps/haproxy-ingress
+kubectl -n haproxy-ingress rollout status --timeout=5m deployment.apps/haproxy-ingress deploy/ingress-default-backend deploy/prometheus
 
 kubectl wait --for condition=established crd/nodeconfigs.scylla.scylladb.com
 kubectl wait --for condition=established crd/scyllaoperatorconfigs.scylla.scylladb.com
 kubectl wait --for condition=established crd/scylladbmonitorings.scylla.scylladb.com
-kubectl wait --for condition=established $( find "${DEPLOY_DIR}/prometheus-operator/" -name '*.crd.yaml' -printf '-f=%p\n' )
+
+if [[ "${SO_INSTALL_PROMETHEUS_OPERATOR}" == "yes" ]]; then
```

Comment:
I'd suggest doing for prometheus what we do for the other manifests, i.e. use SO_PROMETHEUS_OPERATOR_PATH with three states: unset means, empty var means skip deployment, nonempty var means override

Reply:
removed